### PR TITLE
Ruslan's Blog Post

### PR DESCRIPTION
**Pull request by wingedRuslan**
My blog post to introduce myself as a new member of the Whitaker Lab over the summer, GSoC Student :)

### CODE REVIEW

File: _posts/blog/2019-05-20-Ruslan-GSoC.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+> My name is Ruslan Yermakov, I am a senior undergraduate student with a major in Computer Science at Sumy State University, Ukraine.
+I find it so hard to put into words how much I am enthusiastic to be accepted as a Google Summer of Code student 
+and excited to become a member of the Whitaker lab research group and to contribute to the scona project, an open-source toolkit for neuroscientists!
+My interests include data visualization, data science, neuroscience and design of intelligent algorithms and systems.
+
+## Why scona and neuroscience?
+
+Being initiative and enterprising, I am always willing to learn new things and look for new ideas. I value self-development and consider education to be the best investment. 
+Based on the Coursera’s recommendation once I’ve decided to complete the online course “Learning how to learn” (the best ever course) presented by Dr. Barbara Oakley.
```
Comment (KirstieJane) [inline _posts/blog/2019-05-20-Ruslan-GSoC.md]: Can you add a link to the course here?

File: _posts/blog/2019-05-20-Ruslan-GSoC.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+## Why scona and neuroscience?
+
+Being initiative and enterprising, I am always willing to learn new things and look for new ideas. I value self-development and consider education to be the best investment. 
+Based on the Coursera’s recommendation once I’ve decided to complete the online course “Learning how to learn” (the best ever course) presented by Dr. Barbara Oakley. 
+This is how I have got excited about neuroscience field. What I liked the most, all the tips and suggestions of how to “learn smart not hard” were backed up by insightful 
+facts and recent researches, that inform about the way our brain works, which is helpful for improving one's productivity. 
+That’s why the chance to apply my knowledge and skills to develop the tool for neuroscientist was so appealing to me.
+
+From the list of INCF’s projects, Scona project grabbed my attention the most. It matches my previous experience and skills as well as my interests.
```
Comment (KirstieJane) [inline _posts/blog/2019-05-20-Ruslan-GSoC.md]: Add a couple of links. To the INCF list (maybe on neurostars?) and to scona?

Also, I think I always write scona in lowercase to match the package name. Would you mind changing it?

File: _posts/blog/2019-05-20-Ruslan-GSoC.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+I am a happy dog owner, football-addicted (once red, forever red), constantly broadening my outlook by engaging in exciting discussions about the meaning of life and 
+if a hotdog is a sandwich. I am interested in machine learning and artificial intelligence, new advancements and achievements in hightech. Love reading and hearing 
+awesome stories and points of views from interesting people (Elon Musk is not the only one). 
+
+<figure>
+  <img src="/images/Ruslan-GSoC/buddy.jpg"
+       alt="Buddy">
+  <figcaption> Waiting for the summer with my buddy </figcaption>
+</figure>
+
```
Comment (KirstieJane) [inline _posts/blog/2019-05-20-Ruslan-GSoC.md]: Fantastic! I love the picture and this section in general. So lovely to get to know you!

File: _posts/blog/2019-05-20-Ruslan-GSoC.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: Ruslan GSoC Student
+categories: blog
+excerpt: A new member of the Whitaker lab
```
Comment (KirstieJane) [inline _posts/blog/2019-05-20-Ruslan-GSoC.md]: This excerpt shows up in the blog list and so quite a lot of people see it. I think it makes sense to have GSOC in there somewhere.
```suggestion
excerpt: GSOC 2019 student in the lab working on structural covariance networks
```

**Review by KirstieJane — COMMENTED** [review]
This is wonderful @wingedruslan - thank you so much for writing this blog.

There are a couple of comments below, and I wonder if you could please restructure the text so each individual sentence is on a new line. It’s easier to review full sentences, and for version control in general (its easier to track which _sentences_ have changed).

File: _posts/blog/2019-05-20-Ruslan-GSoC.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+Interacting and collaborating with people from The Alan Turing Institute, the UK’s national institute for data science and artificial intelligence, will be incredibly beneficial and valuable for my future. 
+While applying I did not know that project is within a lab at the Turing Institute though. What a great coincidence! 
+
+
+I am delighted of being able to contribute to such a meaningful project, become a member of the Whitaker lab research group and be part of the vast community of INCF organization. 
+Looking forward to the great summer!
+
+<figure>
+  <img src="/images/Ruslan-GSoC/worldCup.jpg"
+       alt="Buddy">
```
Comment (KirstieJane) [inline _posts/blog/2019-05-20-Ruslan-GSoC.md]: I think this alt text needs updating.

File: _posts/blog/2019-05-20-Ruslan-GSoC.md (hunk 1 above)
Comment (wingedRuslan) [reply _posts/blog/2019-05-20-Ruslan-GSoC.md]: Sure, added

File: _posts/blog/2019-05-20-Ruslan-GSoC.md (hunk 2 above)
Comment (wingedRuslan) [reply _posts/blog/2019-05-20-Ruslan-GSoC.md]: Links are added.
From now on I will write "scona" in lowercase :)

**Comment by wingedRuslan** [comment]
Thanks for the review @KirstieJane 

I edited the blog post based on your suggestions :)

File: _data/authors.yml [hunk 6]
```diff
@@ -21,6 +21,13 @@ ang_li: # No spaces please
   avatar: Tongyang.jpeg
   twitter: abc782544421
 
+ruslan_yermakov:
+  name: Ruslan Yermakov
+  email: yermakovruslan@gmail.com
+  bio: "GSoC Student and a member of the Whitaker lab over the summer to contribute to the Scona project"
+  avatar: Ruslan_blogprofile.jpeg
```
Comment (KirstieJane) [inline _data/authors.yml]:
```suggestion
  avatar: Ruslan_blogprofile.jpg
```

**Review by KirstieJane — APPROVED** [review]
There's a tiny typo but everything looks good. You should be able to click the green button and merge once you've accepted that change!

**Pull request merged**